openlibrary/solr/update_work.py

```diff
@@ -510,6 +512,32 @@ def add_list(name, values):
                            for v in e[db_key])
             add_list(solr_key, values)
 
+        # Need this to enable in pytest, which have config.runtime_config = {}
+        conf = config.runtime_config or {
+            'plugin_worksearch': {
+                'has_classifications': True
+            }}
+        if conf.get('plugin_worksearch', {}).get('has_classifications', False):
+            raw_lccs = set(lcc
+                           for ed in editions
+                           for lcc in ed.get('lc_classifications', []))
+            lccs = set(lcc for lcc in map(short_lcc_to_sortable_lcc, raw_lccs) if lcc)
```

Comment:
```suggestion
            lccs = set(short_lcc_to_sortable_lcc(lcc) for lcc in raw_lccs if lcc)
```

Reply:
This slightly different; the result of that function could be `None`, not the input. Marking as won't fix (not an issue).